packages/plugin-autocapture-browser/package.json

```diff
@@ -32,7 +32,7 @@
     "publish": "node ../../scripts/publish/upload-to-s3.js",
     "test": "jest",
     "typecheck": "tsc -p ./tsconfig.json",
-    "version": "yarn version-file && yarn add @amplitude/analytics-client-common@\">=1 <3\"",
+    "version": "yarn version-file && yarn add @amplitude/analytics-client-common@\">=1 <3\" && yarn build",
```

Comment:
@liuyang1520 tagging you since i found potentially the same issue with builds using the wrong version of the package